src/Symfony/Component/Notifier/Bridge/LineBot/LineBotTransport.php

```diff
@@ -42,20 +43,44 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
         }
 
+        $options = $message->getOptions() ?? new LineBotOptions();
+        if (!($options instanceof LineBotOptions)) {
+            throw new InvalidArgumentException('Invalid message provided.');
+        }
+
+        // If there are no messages in the options (for example, if the options are empty),
+        // we should take the message from the notification or subject.
+        if (0 === $options->getMessagesCount()) {
+            if (null !== ($notification = $message->getNotification())) {
+                $options->addMessageFromNotification($notification);
+            } else {
+                $options->addMessage([
+                    'type' => 'text',
+                    'text' => $message->getSubject(),
+                ]);
+            }
+        }
+
+        // If the recipient ID is not set, set it to the default receiver in DSN
+        if (null === $options->getRecipientId()) {
+            $options->to($this->receiver);
+        }
+
+        // If the message is not set, set it to the subject of the message
+        $parameters = $options->toArray();
+        if (empty($parameters['messages'])) {
```

Comment:
We try to avoid using `empty` in Symfony code base, can you replace it with something more precise?